src/lib/ecosystems/resolve-test-facts.ts

```diff
@@ -52,6 +52,10 @@ export async function resolveAndTestFacts(
     : resolveAndTestFactsRegistry(ecosystem, scans, options);
 }
 
+async function getOrgDefaultContext(): Promise<string> {
+  return (await getSelf())?.data.attributes.default_org_context;
```

Comment:
One last thing, was it intentional to add `.data.attributes`, isn't that already taken care of by `getSelf`?

```suggestion
  return (await getSelf())?.default_org_context;
```

Reply:
`getSelf()`  returns a [`SelfResponse`](https://github.com/snyk/cli/blob/7a6c44cfbd1a66b09efbd6001848179f54190420/src/lib/ecosystems/unmanaged/utils.ts#L48) which, as of commit https://github.com/snyk/cli/commit/ff897719447e12990881541b62e44e003122eda0#diff-50795afceb61f2719e051435cf21b7dc96eebb618447a6e1f8fc3df515e0d198, doesn't have a `default_org_context` property (before that commit, the `data.attributes` were moved up, but after that commit, they no longer are); that information is accessed by `.data.attributes.default_org_context`.

Reply:
You are right, I looked at the wrong branch.